packages/sdks/mitosis.config.cjs

```diff
@@ -730,6 +764,8 @@ module.exports = {
     qwik: {
       typescript: true,
       plugins: [
+        // Remove `fetchPriority` attribute from all blocks, since qwik doesn't support it
+        REMOVE_FETCHPRIORITY_PLUGIN,
```

Comment:
What exactly is the error you're facing? I just tried manually adding `fetchPriority` to `img` in the Qwik output and it works:

![CleanShot 2024-06-27 at 15 32 02@2x](https://github.com/BuilderIO/builder/assets/1393142/5ba8cee2-6cf9-4798-bc86-4e5b5f4fa0b3)


![CleanShot 2024-06-27 at 15 31 30@2x](https://github.com/BuilderIO/builder/assets/1393142/af543455-b455-494f-a3b2-6b235160a0c9)

Reply:
oh, it's complaining about it being a custom attribute:

![CleanShot 2024-06-27 at 15 34 54@2x](https://github.com/BuilderIO/builder/assets/1393142/708d7869-d764-432e-b765-3348d07f9fcd)

I think you can swap this for `FETCHPRIORITY_PLUGIN` and it should work for qwik!

Reply:
Oh interesting, hrmm, then we need to handle both `fetchpriority` and `fetchPriority` it looks like. Lemme see if I can just update the qwik mitosis config to use the same plugin I used for react to camel case `fetchpriority`

Reply:
haha jinx we replied at same time

Reply:
no dice
<img width="951" alt="Screenshot 2024-06-27 at 11 41 56 AM" src="https://github.com/BuilderIO/builder/assets/4028730/3999bdc3-65ad-4fc4-8374-71ac364a7945">

Reply:
Ah, I just took another look: this is because you have to upgrade typescript itself to get the latest DOM library HTLMImageElement types.

I'd update only the Qwik SDK's typescript version, to avoid running into other issues if you update the entire monorepo's.